setup.py

```diff
@@ -50,4 +50,6 @@
         },
     test_suite='cnxepub.tests',
     zip_safe=False,
+    dependency_links=[
+                'git+https://github.com/Connexions/cnx-easybake.git#egg=cnx-easybake']
```

Comment:
Just FYI, we'll be removing this during https://github.com/Connexions/cnx-deploy/milestone/3. But for the time being it'll work.